spark3/main.py

```diff
@@ -86,17 +85,11 @@ def parse_trace_to_function(df: DataFrame,
 
         df.sql_ctx.udf.registerJavaFunction("decode_func_%s" % name, DECODE_CONTRACT_FUNCTION_UDF, schema)
 
-        if not contains_column(df.dtypes, "input", "string"):
-            raise ColumnNotFoundInDataFrame("input(string)", df)
```

Comment:
I remove the manual schema validation since DF's exception already contains rich info